animation_nodes/math/euler.pyx

```diff
@@ -0,0 +1,6 @@
+from . number cimport lerp
+
+cdef void mixEul3(Euler3* target, Euler3* a, Euler3* b, float factor):
+    target.x = lerp(a.x, b.x, factor)
+    target.y = lerp(a.y, b.y, factor)
+    target.z = lerp(a.z, b.z, factor)
```

Comment:
Your forgot to set the order here.

Reply:
Should we set the order of `a` as the order of `target` or default order as `XYZ`?

Reply:
Target should have the same order as `a`.

Reply:
Okay.